packages/cli/lib/commands/new.ts

```diff
@@ -100,9 +100,15 @@ command = {
 			Util.warn("Read more at: https://www.infragistics.com/products/ignite-ui-angular/angular/components/general/getting_started.html.", "yellow");
 		}
 
-		if (argv.theme && projectLib.themes.indexOf(argv.theme) === -1) {
+		if (argv.theme && projectLib.themes.findIndex(item => argv.theme.toLowerCase() === item.toLocaleLowerCase()) === -1) {
 			return Util.error("Theme not supported");
 		}
+
+		// check if the theme name starts with a capital letter
+		if (argv.theme[0] !== argv.theme[0].toUpperCase()) {
```

Comment:
I think this will break the jQuery templates' behavior, since the theme names there do not start with a capital letter.
I'd suggest just renaming **all** theme names to be lower case and converting the passed args to lower case as well.